packaging/make_public.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python -B
+#!/usr/bin/python3 -B
```

Comment:
I think while we're at it, I'd like to see this change too:

```suggestion
#!/usr/bin/env python3 -B
```